src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,59 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './CurrentUser.scss';
+import { getUser } from '../../api';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: '',
+  }
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    this.loadData();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.userId !== this.props.userId) {
+      this.loadData();
+    }
+  }
+
+  loadData() {
+    getUser(this.props.userId)
+      .then(users => (
+        this.setState({ user: users.data })
+      ));
+  }
+
+  render() {
+    const { user } = this.state;
+    const { clearUser } = this.props;
+
+    return (
+      <div className="CurrentUser">
+        <h2 className="CurrentUser__title">
+          <span>
+            {`Selected user: ${user.id}`}
+          </span>
+        </h2>
+        <h3 className="CurrentUser__name">{user.name}</h3>
+        <p className="CurrentUser__email">{user.email}</p>
+        <p className="CurrentUser__phone">{user.phone}</p>
+        <button
+          type="button"
+          className="button"
+          onClick={() => {
+            clearUser(0);
+          }}
```

Comment:
```suggestion
          onClick={clearUser}
```
If you'll change your function as I've described above, you can shorten your code here